content/docs/reference/config/helm-customization/index.md

```diff
@@ -0,0 +1,283 @@
+---
```

Comment:
Is "helm customization" the right name for this file? This is customizing the installation options, not customizing helm, right? Perhaps "installation-options" would be a better name?